### DISCUSSION TITLE
Desktop user, Linux

### DISCUSSION THREAD

**Discussion by blaineclrk** [EFForg/privacybadger#2766] [Q&A]
I just saw webext-privacy-badger in Synaptic. I've looked over some documentation, but it's confusing. How, or can I, use this on my laptop? Will WPB monitor all web traffic for all internet software in this type of installation or does it have to be directed at specific software or traffic?

**Comment by ghostwords** [comment]
Hello!

Privacy Badger is a browser extension that automatically learns to block invisible trackers. As a browser extension, Privacy Badger works in your browser only.

To use it, visit [Privacy Badger's homepage](https://privacybadger.org/) and click the big "install" button.

To learn more about Privacy Badger, see the [Frequently Asked Questions section](https://privacybadger.org/#faq) (under the "install" button).

I do not recommend using the Ubuntu package, as it appears to be out of date. Installing from the [homepage](https://privacybadger.org/) will get you the latest version and will also automatically keep you up to date.

Let me know if you have any other questions.
